doc/design/dlt_offline_logstorage_design.md

```diff
@@ -0,0 +1,743 @@
+# DLT Offline Logstorage Design Specification
+
+Back to [README.md](../../README.md)
+
+## Executive Summary
+
+This document describes DLT Offline Logstorage feature. It includes software
+architecture and design. DLT Offline Logstorage offers the possibility to store
+application logs on different storage devices by just plug-in the storage device
+into the target. The configuration of what log messages will be stored is read
+from a special configuration file which has to be installed on the storage
+device before inserting into the target.
+
+Usage is available in [dlt_offline_logstorage.md](../dlt_offline_logstorage.md).
+
+## Scope
+
+This document is written to understand design and implementation, and should
+help to use DLT Logstorage feature in the correct way.
+
+## Requirements
+
+Requirements for the DLT Message Filter component are written down in
+[JAMA](https://jama.adit-jv.com).
```

Comment:
Is message filter part of this upstream commit? 
Please remove the link to adit's jama.

Reply:
Hello @michael-methner ,
Please view the rework version!